backend/pick-git/src/test/java/com/woowacourse/pickgit/integration/post/PostServiceIntegrationTest.java

```diff
@@ -153,6 +154,28 @@ void write_LoginUser_Success() {
         assertThat(responseDto.getId()).isNotNull();
     }
 
+    @DisplayName("사용자는 게시글 등록시 중복된 태그를 작성할 수 없다.")
+    @Test
+    void write_LoginUserWithDuplicateTag_Fail() {
+        // given
+        PostRequestDto requestDto = PostRequestDto.builder()
+            .token(ACCESS_TOKEN)
+            .username(USERNAME)
+            .images(
+                List.of(
+                    FileFactory.getTestImage1(),
+                    FileFactory.getTestImage2()
+                )
+            )
+            .githubRepoUrl(githubRepoUrl)
+            .tags(List.of("Java", "Javascript", "Java"))
+            .content(content).build();
+
+        // when, then
+        assertThatCode(() -> postService.write(requestDto))
+            .isInstanceOf(CannotAddTagException.class);
```

Comment:
```suggestion
        assertThatCode(() -> postService.write(requestDto))
            .isInstanceOf(CannotAddTagException.class)
            .hasFieldOrPropertyWithValue("errorCode", "XXX")
            .hasFieldOrPropertyWithValue("httpStatus", "XXX")
            .hasMessage("XXX");
```

이렇게 발생하는 예외의 에러 코드, 상태 코드, 메시지까지 확인할 수 있어요 !